CHANGES.md

```diff
@@ -15,6 +15,7 @@ unreleased
       of the involved data-structures. (#1889)
   + test suite
     - Add a test case illustrating wrong open order proposed in issue #1900. (#1901)
+    - Add a test for outline generation in .mli. (#1933)
```

Comment:
I don't think that the branch needs a CHANGE entry

Reply:
Fixed.